tests/e2e/manifest/send_traffic.sh

```
@@ -105,9 +120,9 @@ for i in $(seq 1 $MAX_RETRIES); do
     echo "\n${CYAN}--- Request $i/$MAX_RETRIES ---${NC}"
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)